driverPath: /dev/spidev0.0
alwaysOn: true
spiMode: 0
spiSpeed: 1000000
spiBits: 8
frequency: 10.0

maxFreq: 10.0
minFreq: 0.2

field_of_view: 0.01
minRange: 0.0
maxRange: 0.80
channelNo: [0, 1, 2, 3]

parK: 0.42
parA: 0.0000473
parB: -0.0013461
